src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -28,9 +26,7 @@ if [%BUILDTOOLS_USE_CSPROJ%]==[] (
 { ^
   "dependencies": ^
     { ^
-      "MicroBuild.Core": "%MICROBUILD_VERSION%", ^
-      "Microsoft.Portable.Targets": "%PORTABLETARGETS_VERSION%", ^
-      "Microsoft.Net.Compilers": "%ROSLYNCOMPILERS_VERSION%" ^
+      "MicroBuild.Core": "%MICROBUILD_VERSION%" ^
```

Comment:
We still need the microbuild stuff but we should move it to the toolruntime project instead.